package.json

```diff
@@ -229,6 +228,7 @@
     "yargs": "^17.5.1"
   },
   "optionalDependencies": {
+    "couchbase": "^4.3.0",
```

Comment:
Is there a reason to chenge this to optional dependencies?

Reply:
> Does no longer install with Node v14

:)